server/lib/utils/util.go

```diff
@@ -50,3 +53,48 @@ func IFIP(name string, need_ipv6 bool) (net.IP, error) {
 	}
 	return nil, fmt.Errorf("interface %v: no suitable setting found", name)
 }
+
+type WaitGroup struct {
```

Comment:
__[gometalinter]__
* [golint] exported type WaitGroup should have comment or be unexported